frontend/src/modules/layout/config/links/home.ts

```diff
@@ -0,0 +1,12 @@
+import { MenuLink } from '@/modules/layout/types/MenuLink';
+
+const home: MenuLink = {
+  id: 'home',
+  label: 'Home',
```

Comment:
There are some designs where Nuno as Dashboard with a different icon. Can you confirm with me which one is the correct one?

Reply:
Below in designs the one with border around are correct ones and the icon is correct